readme.md

```diff
@@ -25,6 +25,47 @@ This runs:
 * legacy integration tests written in cucumber
 * revised integration tests for the updated UKHPI app, running under MiniTest
 
+## Running the tests in a Docker container
```

Comment:
The ssh tunnel is horrid. Correct the security groups rather than kludging a hole through.

Reply:
You may care to think that :)

Firstly, that's a value judgement, not a substantive comment of a problem.

Secondly, updating the security access controls is beyond the scope of the work for this pull request.  I hear what you say - but I think that updating the network, security controls etc should be subject of separate ticket.  Maybe we could have chat about alternatives and put such a ticket together?

Bear in mind, this was designed so that I could fix problems form a service station part way up the M6 at a time when I didn't have a working vpn to court lodge.